### PR TITLE
setup docsify

### PR DESCRIPTION
**Pull request by AlexNiallMalin**
@mcollina  Let me know if there's anything else to add.

### CODE REVIEW

**Comment by mcollina** [comment]
I think this needs a bit more work. The panel on the left is mostly empty, and I think this could use a bit of navigation.

If you would like to move most of the README into individual files in the docs folder, go for it. The structure that pino follows is a good one: https://github.com/pinojs/pino.

Screenshot 2020-10-08 at 15 02 16

**Comment by AlexNiallMalin** [comment]
Sure thing! I wanted to keep it nice and simple then see where you wanted to take it from there.
Ill have a look at that and also look over PR #312 and see if theres anything I must do.
Do you have a logo you would like me to add and/or a colour scheme you wish to use?

**Comment by mcollina** [comment]
We have no logo at the minute unfortunately :/.

**Comment by AlexNiallMalin** [comment]
Ah the readme file has changed since. I'll look at this ASAP and try and docsify it. Sorry for the delay, i've been in the process of securing a job recently.

**Comment by mcollina** [comment]
Can you please rebase?

**Comment by AlexNiallMalin** [comment]
Hey @mcollina I have rebased and also gone through and resolved any style consistency issues between this branch and the branch that was merged. All of the additional content from that branch is now integrated into the docsify implementation.

**Comment by PabloSzx** [comment]
Why is almost everything inside https://github.com/AlexNiallMalin/mercurius/blob/docsify/docs/example.md ?

The main reason of separating the docs is to ease the navigation and example.md doesn't even have TOC

File: docs/protocol-extension.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ The GraphQL over WebSocket Protocol used by `mercurius` follows apollo's protoco
 ## Message structure extension
 
 This is the extended structure of each message:
-
-```ts
+```
```
Comment (PabloSzx) [inline docs/protocol-extension.md:8]: There is no reason to lose the syntax highlighting

File: docs/protocol-extension.md [hunk 2]
```diff
@@ -22,17 +21,16 @@ export interface OperationExtension {
 ```
 
 ## Server -> Server
-
 ### `connectionInit` extension
 
 Gateway uses this extension to share the `connection_init` payload with a service when the connection is already established between gateway and services.
 
-```ts
+```
```
Comment (PabloSzx) [inline docs/protocol-extension.md:28]: There is no reason to lose the syntax highlighting

**Comment by AlexNiallMalin** [comment]
Oh i do apologise, my mistake. #323 looks really good!

**Pull request closed without merging**